packages/atlas-react/src/components/BasicButton/BasicButton.scss

```diff
@@ -1,7 +1,159 @@
+  --#{$namespace}-button-background-color-active: #{color(blue-200)};
+  --#{$namespace}-button-color: #{color(blue-400)};
+}
+
+.BasicButton.BasicButton--tertiary {
+  --#{$namespace}-button-background-color: #{color(light-gray-100)};
+  --#{$namespace}-button-background-color-hover: #{color(light-gray-400)};
+  --#{$namespace}-button-background-color-focus: #{color(light-gray-400)};
+  --#{$namespace}-button-background-color-focus-shadow: #{color(blue-200)};
+  --#{$namespace}-button-background-color-active: #{color(black-400)};
+  --#{$namespace}-button-color: #{color(black-400)};
+}
+
+.BasicButton.BasicButton--destructive {
+  --#{$namespace}-button-background-color: #{color(red-400)};
+  --#{$namespace}-button-background-color-hover: #{color(red-500)};
+  --#{$namespace}-button-background-color-focus: #{color(red-500)};
+  --#{$namespace}-button-background-color-focus-shadow: #{color(red-100)};
+  --#{$namespace}-button-background-color-active: #{color(red-500)};
+  --#{$namespace}-button-color: #{color(white-400)};
+}
+
+.BasicButton.BasicButton--size-default  {
+  --#{$namespace}-button-padding: #{spacer(mini)} #{spacer(small)};
+  --#{$namespace}-button-height: 36px;
+  --#{$namespace}-button-font-size: var(--a-size-font-body-14);
+  --#{$namespace}-button-width: auto;
+  --#{$namespace}-button-leading-icon-margin: 0 #{spacer(mini)} 0 0;
+  --#{$namespace}-button-trailing-icon-margin: 0 0 0 #{spacer(mini)};
+}
+
+.BasicButton.BasicButton--size-full-width  {
+  --#{$namespace}-button-padding: #{spacer(mini)} #{spacer(small)};
+  --#{$namespace}-button-height: 36px;
+  --#{$namespace}-button-font-size: var(--a-size-font-body-14);
+  --#{$namespace}-button-width: 100%;
+  --#{$namespace}-button-leading-icon-margin: 0 #{spacer(mini)} 0 0;
+  --#{$namespace}-button-trailing-icon-margin: 0 0 0 #{spacer(mini)};
+}
+
+.BasicButton.BasicButton--size-small  {
+  --#{$namespace}-button-padding: #{spacer(micro)} calc(#{spacer(micro)} + #{spacer(mini)});
+  --#{$namespace}-button-height: 28px;
+  --#{$namespace}-button-font-size: var(--a-size-font-body-14);
+  --#{$namespace}-button-width: auto;
+  --#{$namespace}-button-leading-icon-margin: 0 #{spacer(micro)} 0 0;
+  --#{$namespace}-button-trailing-icon-margin: 0 0 0 #{spacer(micro)};
+}
+
+.BasicButton.BasicButton--size-mini  {
+  --#{$namespace}-button-padding: #{spacer(micro)} #{spacer(mini)};
+  --#{$namespace}-button-height: 26px;
+  --#{$namespace}-button-font-size: var(--a-size-font-body-12);
+  --#{$namespace}-button-width: auto;
+  --#{$namespace}-button-leading-icon-margin: 0 #{spacer(micro)} 0 0;
+  --#{$namespace}-button-trailing-icon-margin: 0 0 0 #{spacer(micro)};
+}
+
+.BasicButton.BasicButton--align-left {
+  --#{$namespace}-button-align: flex-start;
+}
+
+.BasicButton.BasicButton--align-right {
+  --#{$namespace}-button-align: flex-end;
+}
+
+.BasicButton.BasicButton--align-center {
+  --#{$namespace}-button-align: center;
+}
+
+.BasicButton {
```

Comment:
there's some cleanup that could be done in this file but I'm really hoping for the chance to revert this in January so I'm not worried about it right now

Reply:
sounds good.